### PR TITLE
[C-2601 C-2627 C-2628 C-2638] Playlist library QA

### PR DESCRIPTION
**Pull request by dylanjeffers**
### Description

Addresses majority of callouts in playlist-library qa
Note this should be reviewed after playlist create flow is merged, as depends on it.

### TODO
drag-drop new playlists, and general playlist-library sync management coming in another pr

### CODE REVIEW

**Comment by audius-infra** [comment]
Preview this change https://demo.audius.co/dj-c-2601-left-nav-qa-1

File: packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistFolderNavItem.tsx [hunk 1]
```diff
@@ -51,7 +54,14 @@ const messages = {
 export const PlaylistFolderNavItem = (props: PlaylistFolderNavItemProps) => {
   const { folder, level } = props
   const { name, contents, id } = folder
-  const hasUpdate = false
```
Comment (Kyle-Shanks) [inline packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistFolderNavItem.tsx:54]: lol

Comment (dylanjeffers) [reply packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistFolderNavItem.tsx:54]: shhhhh, nothing to see here 🗡️

File: packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistUpdateDot.module.css [hunk 2]
```diff
@@ -1,6 +1,6 @@
 .root {
   position: absolute;
-  left: calc(32px - 10px);
+  left: calc(var(--unit-8) - var(--unit-3));
```
Comment (Kyle-Shanks) [inline packages/web/src/components/nav/desktop/PlaylistLibrary/PlaylistUpdateDot.module.css:3]: 🧐

**Review by Kyle-Shanks — APPROVED** [review]
good stuff

**Pull request merged**